pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -60,6 +63,7 @@ func NewCmdStartBuild(fullName string, f *clientcmd.Factory, out io.Writer) *cob
 	}
 	cmd.Flags().String("from-build", "", "Specify the name of a build which should be re-run")
 	cmd.Flags().Bool("follow", false, "Start a build and watch its logs until it completes or fails")
+	cmd.Flags().Bool("wait-for-complete", false, "Wait for a build to complete (will return error if build fails)")
```

Comment:
just an cosmetic suggestion :) , nuke the parentheses and rather use the semicolon as the example bellow.